CONTRIBUTING.md

```diff
@@ -81,6 +81,15 @@ Our merge strategy relies on the classification of PRs on two axes:
 * How controversial are the design decisions
 * How complex is the implementation
 
+Each issue [label](https://github.com/bevyengine/bevy/labels) has a prefix denoting its category:
```

Comment:
```suggestion
Each issue and pull request [label](https://github.com/bevyengine/bevy/labels) has a prefix denoting its category:
```

Reply:
or just "each label"? it's what they're called in GitHub docs and UI